.mypy.ini

```diff
@@ -9,3 +9,6 @@ ignore_errors: True
 # Turn off mypy for unit tests
 [mypy-*.tests.*]
 ignore_errors: True
+
+plugins =
+    mypy_drf_plugin.main
```

Comment:
Is this required? What happens if you don't have it?

Reply:
In the original repo, the following is mentioned: "To make mypy aware of the plugin, you need to add to add it in the mypy.ini".  I guess even if the plugin is listed as a dependency in the pre-commit hook, it won’t be used unless activated via the mypy.ini. Or if it activated it might perform less accurate type checking. 

I tried to delete it from mypy.ini and I catch the same errors. Still investigating it.

Reply:
Would you mind sending a link to the "original repo"?

Reply:
Here you are: https://github.com/typeddjango/djangorestframework-stubs/tree/master

Reply:
Good, let's assume it is mandatory :)